AEPCore/Sources/eventhub/history/EventHistoryDatabase.swift

```diff
@@ -61,6 +61,7 @@ class EventHistoryDatabase {
         dispatchQueue.async {
```

Comment:
not your code, but can we add a capture of `[weak self]` for any of these async blocks where we are using `self` so we can prevent potential leaking?